elfi/methods/inference/bolfire.py

```diff
@@ -443,8 +413,64 @@ def _resolve_acquisition_method(self, acquisition_method):
             return acquisition_method
         raise TypeError('acquisition_method must be an instance of AcquisitionBase.')
 
+    def _init_round(self):
+        """Initialize data collection round."""
+        # Initialize collected data
+        self._likelihood = np.array([]).reshape(0, self.data_dim)
```

Comment:
Would it be possible to implement another type of `_likelihood` construction than concatenating arrays on top of the empty array?